.codecov.yml

```diff
@@ -5,13 +5,13 @@ coverage:
       mobile:
         paths: "packages/mobile/"
         flags: mobile
-        threshold: 40%
-        target: 10%
+        threshold: 5%
+        target: 70%
       web:
         paths: "packages/web/"
         flags: web
-        threshold: 5%
-        target: 70%
+        threshold: 2%
```

Comment:
from my understanding threshold is how much it can descrease from master before a flag is raised, and target is the absolute amount of lines that need to be tested overall